lang/en.json

```diff
@@ -144,8 +144,8 @@
         "or": "or",
         "compare": "Compare",
         "sign_up": "Sign Up",
-        "login": "Login",
-        "logout": "Logout",
+        "login": "Log in",
```

Comment:
+1 for "Sign in".

From previous reading and discussion, I think the best practice in this space is to use "Sign in" / "Sign out" / "Register" (not "Sign up").